### PR TITLE
Add action IDs to the color selection commands

### LINKED ISSUE #17819: Updated to 1.22.2362.0, and the “Color selection, foreground” commands are MIA
### Windows Terminal version

1.22.2362.0

### Windows build number

10.0.26277.5000

### Other Software

_No response_

### Steps to reproduce

Select some text, open command palette, type "color selection".

### Expected Behavior

Expected to see about 20 matching commands: 0-9 for "Color selection, foreground: ..." and 0-9 for "Color selection background: ...".

### Actual Behavior

I only see about 10--just the "background" ones.

The color selection background commands work, at least... except the ctrl+shift+0 shortcut does not work. :(

### PR DESCRIPTION
**Pull request by PankajBhojwani**
## Summary of the Pull Request
Add action IDs to the default commands for color selection

## Validation Steps Performed
Color selection commands now show up in the command palette

## PR Checklist
- [x] Closes #17819
- [ ] Tests added/passed
- [ ] Documentation updated
   - If checked, please file a pull request on [our docs repo](https://github.com/MicrosoftDocs/terminal) and link it here: #xxx
- [ ] Schema updated (if necessary)

### CODE REVIEW

**Review by jazzdelightsme — APPROVED** [review]
:shipit:

**Review by lhecker — APPROVED** [review]
🎉

**Review by carlos-zamora — APPROVED** [review]

**Review by zadjii-msft — APPROVED** [review]

**Comment by DHowett** [comment]
Marked up for 1.22 - thanks for the report and for the quick fix!

File: src/cascadia/TerminalSettingsModel/enableColorSelection.json [hunk 1]
```diff
@@ -364,7 +403,8 @@
                 "foreground": "i00",
                 "background": "i03"
             },
-            "keys": "ctrl+shift+0"
+            "keys": "ctrl+shift+0",
```
Comment (DHowett) [inline src/cascadia/TerminalSettingsModel/enableColorSelection.json:406]: Hey, does having `keys`, `command` and `id` in the same settings file -- even though it is a SYSTEM settings file -- result in us changing the user's `keys` list? (e.g. do we accidentally up-convert these key/command/id triples into key/id and action/id pairs?)

Comment (PankajBhojwani) [reply src/cascadia/TerminalSettingsModel/enableColorSelection.json:406]: We do convert these into key/id and action/id pairs in our internal maps, but they're still in the `InBox` layer so they don't show up in the user's settings file

**Pull request merged**